<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrarVendedorDialog</class>
 <widget class="QDialog" name="RegistrarVendedorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrar un Vendedor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="usuarioLabel">
       <property name="text">
        <string>     Usuario : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usuarioLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="claveLabel">
       <property name="text">
        <string>      Contraseña:  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="claveLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>    Repite la contraseña: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="repiteClaveLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>252</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBoxIsAdmin">
     <property name="text">
      <string>Agregar como Administrador</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBoxRegistrar">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
